types/helpers/helpers.options.d.ts

```diff
@@ -0,0 +1,50 @@
+import { IFontSpec } from '../core/interfaces';
+
+export interface ICanvasFontSpec extends IFontSpec {
+  string: string;
+}
+/**
+ * Parses font options and returns the font object.
+ * @param {object} options - A object that contains font options to be parsed.
+ * @return {object} The font object.
+ */
+export function toFont(options: Partial<IFontSpec>): ICanvasFontSpec;
+
+/**
+ * Converts the given line height `value` in pixels for a specific font `size`.
+ * @param {number|string} value - The lineHeight to parse (eg. 1.6, '14px', '75%', '1.6em').
+ * @param {number} size - The font size (in pixels) used to resolve relative `value`.
+ * @returns {number} The effective line height in pixels (size * 1.2 if value is invalid).
+ * @see https://developer.mozilla.org/en-US/docs/Web/CSS/line-height
+ * @since 2.7.0
+ */
+export function toLineHeight(value: string, size: number): number;
+
+/**
+ * Converts the given value into a padding object with pre-computed width/height.
+ * @param {number|object} value - If a number, set the value to all TRBL component;
+ *  else, if an object, use defined properties and sets undefined ones to 0.
+ * @returns {object} The padding values (top, right, bottom, left, width, height)
+ * @since 2.7.0
+ */
+export function toPadding(
+  value: number | { top: number; left: number; right: number; bottom: number }
```

Comment:
Shouldn't all these input properties be optional?

Reply:
Good point. `top`, etc are all defaulted to 0 if falsy